### DISCUSSION TITLE
Mame Emulator not responding to Antimicrox commands

### DISCUSSION THREAD

**Discussion by Gazenbeek** [AntiMicroX/antimicrox#544] [Q&A]
I have set up 1 button on a xbox one x controller in antimicro to press the keyboard P key.

In notepad it works fine.

when i start mame emulator it wont detect the keypress P

windows 10 64 bit
mame 64 latest
antimicrox latest

**Comment by pktiuk** [comment]
In this case, I think something is wrong with Mame emulator.    
If systems sees key presses correctly
